src/main/java/org/broadinstitute/hellbender/tools/exome/germlinehmm/CopyNumberTriStateTransitionProbabilityCache.java

```diff
@@ -137,8 +137,16 @@ public double visit(final int to, final int from, final double probability) {
 
     /**
      * Wrapper for a {@link RealMatrix} that is indexed by the CopyNumberTriState enum
+     *
```

Comment:
I have no idea either; private static inner classes in Java behave differently than those in saner OOP languages :) in any event, we don't care about this as all TriState stuff will be dead code soon.